crates/tauri-plugin-holochain-service-consumer/src/mobile.rs

```diff
@@ -70,4 +76,16 @@ impl<R: Runtime> HolochainServiceConsumer<R> {
 
         Ok(window_builder)
     }
+
+    /// Setup a holochain app, then create the main tauri window builder that injects the holochain env
+    pub fn setup_app_main_window(
```

Comment:
Function that the actual consumer tauri app will call: sets up the holochain app and builds the main window with __HC_LAUNCHER_ENV__ injected.